Automattic/wp-calypso
# Reader: add sort to tags page

Part of pe7F0s-1gP-p2.

Adds a filter to filter the tag page by date. As suggested in D122845-code I've used the same endpoint as discover for the popular tags page.

I used the parameter names "sort=relevance" and "sort=date" to match the /read/search page.

Note that the mobile design is not ideal, the search page uses a fixed position are to achieve a good result, we may need to rework this page a bit.

Clicks are tracked.

Default sort is by date.

### Testing instructions
Apply D123287-code and sandbox public-api.wordpress.com
go to the /tag/:tag pages
click the buttons to sort by date or relevance, results for relevance should match the discover page. clicks should create requests to tracks
